### ISSUE TITLE
how to get the 3d coordinates after post process

### ISSUE THREAD

**Issue by Xinxinatg** [UnveilStudio/SharedToxs_Unveil_Studio#1]
Hey thanks for the work you have done! I am wondering how you tried to export the 3d coordinates after calibrating? I think the 3d coordinates would be more helpful if they are calibrated in the way that the feet will not go below the floor.

**Comment by UnveilStudio** [comment]
Hi ,

I do not understand to what you are talking about. Can you please be more precise? Do you mean the maxine sdk? Let me know how i can help. We can schedule a call if you want. Regards, Simone

**Comment by Xinxinatg** [comment]
Hey Simone:

Thanks heaps for your reply! This is a example of the pose tracking result [with](https://photos.app.goo.gl/fhhEABfSHYzFNkTF8) and [without](https://photos.google.com/share/AF1QipO8YCIkG3qrHo8oN7S7mOGUSUwqLsxVXbGGLQkr_t5u9Q0iVCTnfNOXfXUjXSDckg/photo/AF1QipNAFZ8WGgYssYC15djhJZh40DB0R3dodtFCS62q?key=NjVMVlBXZGZEaGM5Uk5fdWlzNGdTbDVQYjQ2T1R3) calibratio. You can see that without calibration the feet literally sank into the ground which is not possible in real life. I am hoping to get the 3d coordinates information of the skeletons to train my deep learning model, so it is helpful to get the 3d coordinates information as realistic as possible. 

I assume you are located in Germany? I am in sydney which is nine hours ahead of you. I will be available for next 1 hour or it would be great you can propose a time which would work for you.

Best regards,
Tim

**Comment by UnveilStudio** [comment]
Hi ,

The pose tracker is taken from machinima pose tracker. From the omniverse suite.(the nvdia free software bundle). I made a tutorial about it: https://youtu.be/N9OVtcWJHVU. The example works fine in TD since i do not need any precise order. To invert it. You could pack each set of joints in a another list and invert the result.

**Comment by Xinxinatg** [comment]
Thanks a lot! I found this repo via your youtube channel actully. But let me try again with the method you present in the tutorial
